code/game/data_huds.dm

```diff
@@ -183,22 +183,30 @@
 	var/image/infection_hud = hud_list[XENO_EMBRYO_HUD] //State of the xeno embryo.
 	var/image/simple_status_hud = hud_list[STATUS_HUD_SIMPLE] //Status for the naked eye.
 	var/image/xeno_reagent = hud_list[XENO_REAGENT_HUD] // Displays active xeno reagents
-	var/static/image/neurotox_image = image('icons/mob/hud.dmi', src, "neurotoxin")
-	var/static/image/hemodile_image = image('icons/mob/hud.dmi', src, "hemodile")
-	var/static/image/transvitox_image = image('icons/mob/hud.dmi', src, "transvitox")
+	var/image/neurotox_image = image('icons/mob/hud.dmi', src, "neurotoxin")
+	var/image/hemodile_image = image('icons/mob/hud.dmi', src, "hemodile")
+	var/image/transvitox_image = image('icons/mob/hud.dmi', src, "transvitox")
```

Comment:
instead of changing those cache the others too, everytime you use image() you're creating a new appearance/image, making it so everytime you call this proc you make new images as opposed to cache'ing